src/overlays/actors/ovl_Dm_An/z_dm_an.c

```diff
@@ -160,7 +160,7 @@ Actor* func_80C1C8E8(PlayState* play) {
         }
 
         tempActor = foundActor->next;
-        if (tempActor == NULL || NULL) {
+        if (tempActor == NULL || false) {
```

Comment:
Oh good grief, I suppose this is required? Wonder if it comes from a typo.

Reply:
Yeah, it's required. It can either be `false` or `NULL` (or `0` I guess). I just wanted to make sure it was synchronized between the two files.

Reply:
False seems more correct here.

Reply:
Yeah, I agree. That's why I updated DmAn to use `false` like DmGm already did.